suites/pacific/rgw/tier-2_sse_s3_encryption.yaml

```diff
@@ -150,10 +150,6 @@ tests:
       polarion-id: CEPH-83574040
   - test:
       config:
```

Comment:
is there any reason we have removed bellow config?
run-on-rgw: true
        extra-pkgs:

Reply:
its unnecessary for the test
I have added in the description also, pls check

please let me know if Iam wrong and if it is actually required

Reply:
@anrao19 
reverted the change
please re-review